cargo-shuttle/src/client.rs

```diff
@@ -37,7 +37,8 @@ impl ShuttleApiClient {
                     )]))
                     .unwrap(),
                 )
-                .timeout(Duration::from_secs(60))
+                // some operations, like project delete, can take a while.
+                .timeout(Duration::from_secs(60 * 5))
```

Comment:
I'm not sure if we should increase the timeout across the board. If we have some request that isn't making progress, the user would need to wait a lot longer before they get a failure. I suppose the alternative is refactoring the project delete call to polling, is that going to be too time-consuming to implement?

Reply:
Or I set the high timeout for the delete request only.

Reply:
Nice, thanks!